src/index.scss

```diff
@@ -1,8 +1,13 @@
+@import 'carbon-components/scss/globals/scss/styles.scss';
 /* Remove overrides once Carbon bugs are fixed upstream. */
 
 /*
 The React <Content /> component uses the `main` element which IE11 doesn't recognize
 
+$feature-flags: (
+  ui-shell: true,
+);
+
 @link https://github.com/carbon-design-system/carbon/issues/2789
```

Comment:
The feature flags declaration is needed and should come before the Carbon styles import